### PR TITLE
Issue #5: Update Graduation Dates

### PR DESCRIPTION
**Pull request by eddiechoi00**
### Feature Request
Changed graduation years.

## Additional Notes
Please ignore any of the other changes I made to accommodate running this locally. If you can't accept/merge the pull request as needed, you might have to make these changes (graduation years) manually; although it's a simple change I apologize for the inconvenience.

### CODE REVIEW

**Comment by jackheuberger** [comment]
Closes #5

File: Makefile [hunk 1]
```diff
@@ -16,6 +16,7 @@ create-env:
 reset-data:
 	docker-compose down || true
 	docker volume remove myhackwashu_data || true
+	docker-compose down --volumes || true
```
Comment (jackheuberger) [inline Makefile:19]: Can you switch lines 18 and 19? Volumes should be brought down before being removed

File: app/hackathon_variables.py [hunk 2]
```diff
@@ -22,7 +22,8 @@
 elif not HACKATHON_DOMAIN:
     HACKATHON_DOMAIN = 'localhost:80'
 # Hackathon contact email: where should all hackers contact you. It will also be used as a sender for all emails
-HACKATHON_CONTACT_EMAIL = os.environ.get('DEV_EMAIL', 'organizer@hackwashu.io')
+# HACKATHON_CONTACT_EMAIL = os.environ.get('DEV_EMAIL', 'organizer@hackwashu.io')
+HACKATHON_CONTACT_EMAIL = 'epsilon41@naver.com'
```
Comment (jackheuberger) [inline app/hackathon_variables.py:26]: Remove this file from commit

File: applications/migrations/0040_auto_20220624_1615.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.7 on 2022-06-24 21:15
+
+from django.db import migrations
+import multiselectfield.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```
Comment (jackheuberger) [inline applications/migrations/0040_auto_20220624_1615.py:9]: Remove this file from commit

File: applications/migrations/0041_auto_20220624_1626.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.7 on 2022-06-24 21:26
```
Comment (jackheuberger) [inline applications/migrations/0041_auto_20220624_1626.py:1]: Remove this file from commit

File: applications/models.py [hunk 5]
```diff
@@ -108,7 +108,7 @@
 EXTRA_NAME = [' 2016 Fall', ' 2016 Winter', ' 2017 Fall', ' 2017 Winter', ' 2018', ' 2019', '2021']
 PREVIOUS_HACKS = [(i, HACK_NAME + EXTRA_NAME[i]) for i in range(0, len(EXTRA_NAME))]
 
-YEARS = [(int(size), size) for size in ('2020 2021 2022 2023 2024 2025 2026 2027'.split(' '))]
+YEARS = [(int(size), size) for size in ('2022 2023 2024 2025 2026'.split(' '))]
```
Comment (jackheuberger) [inline applications/models.py:111]: Are the frontend selection options generated automatically off of this list?

**Review by jackheuberger — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (eddiechoi00) [reply Makefile:19]: Yes of course.

File: app/hackathon_variables.py (hunk 2 above)
Comment (eddiechoi00) [reply app/hackathon_variables.py:26]: Yessir

File: applications/models.py (hunk 5 above)
Comment (eddiechoi00) [reply applications/models.py:111]: Yes I checke
Screen Shot 2022-06-25 at 7 12 39 PM
d

File: applications/migrations/0040_auto_20220624_1615.py (hunk 3 above)
Comment (eddiechoi00) [reply applications/migrations/0040_auto_20220624_1615.py:9]: The entire file?

**Review by jackheuberger — APPROVED** [review]
lgtm!

**Pull request merged**